views/default/elgg/lightbox.js

```diff
@@ -114,6 +114,14 @@ define('elgg/lightbox', function (require) {
 		 */
 		close: function () {
 			$.colorbox.close();
+		},
+		
+		/**
+		 * Resizes the colorbox
+		 * @return void
+		 */
+		resize: function () {
```

Comment:
I'd be OK adding that and squeezing it in 2.2. Better than folks continuing to call colorbox API directly.